### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by KillianCourvoisier**
Update: Starting needs

### CODE REVIEW

File: src/tutorials/konnector/index.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ To protect your data, each connector runs inside a container in order to sandbox
 
 > ⚠️ For historical reasons, in the Cozy codebase, a cozy connector is named konnector, please follow this convention if modifying an existing application.
 
+What will you need to start ?
+
+- Node 16, you can check your version with `node --version` in your shell.
+- Yarn, you can check the version with `yarn --version` in your shell.
+- [ACH](https://docs.cozy.io/en/ach/)
```
Comment (Crash--) [inline src/tutorials/konnector/index.md:12]: Not sure we need ACH to start. We should remove this line

File: src/tutorials/konnector/index.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ To protect your data, each connector runs inside a container in order to sandbox
 
 > ⚠️ For historical reasons, in the Cozy codebase, a cozy connector is named konnector, please follow this convention if modifying an existing application.
 
+What will you need to start ?
+
+- Node 16, you can check your version with `node --version` in your shell.
```
Comment (Crash--) [inline src/tutorials/konnector/index.md:10]: Not sure it's relevant to say how to check the node version. Maybe you can add a link to nodejs website

File: src/tutorials/konnector/index.md [hunk 3]
```diff
@@ -5,6 +5,12 @@ To protect your data, each connector runs inside a container in order to sandbox
...
+- Yarn, you can check the version with `yarn --version` in your shell.
```
Comment (Crash--) [inline src/tutorials/konnector/index.md:11]: Same here, we should add a link to how to install yarn. Also we should add that we need Yarn V1 and not the v2 for now.

File: src/tutorials/konnector/index.md [hunk 4]
```diff
@@ -5,6 +5,11 @@ To protect your data, each connector runs inside a container in order to sandbox
 
 > ⚠️ For historical reasons, in the Cozy codebase, a cozy connector is named konnector, please follow this convention if modifying an existing application.
 
+What will you need to start ?
+
+- [Node](https://nodejs.org/en/) (16), follow the link to [nodejs](https://nodejs.org/en/docs/) doc for proper installation. When it's done, you can check your version with `node --version` in your shell.
+- [Yarn](https://yarnpkg.com/), again follow the [Yarn doc](https://yarnpkg.com/getting-started) for proper install. Check the version with `yarn --version` in your shell.
```
Comment (Crash--) [inline src/tutorials/konnector/index.md:11]: https://classic.yarnpkg.com/lang/en/ this is the right link for Yarn since we don't use yarn 2 ATM

Comment (KillianCourvoisier) [reply src/tutorials/konnector/index.md]: Changes done ✅

**Pull request merged**